index.bs

```diff
@@ -10068,7 +10071,8 @@ Attributes</h5>
 		When an unhandled exception is thrown from the processor's
 		<code>constructor</code>, <code>process</code> method,
 		or any user-defined class method, the processor will
-		<a>queue a task</a> to fire an
+		<a>queue a task</a> to
+		<a spec="dom" lt="fire an event">fire</a> an
```

Comment:
I was pretty serious about the DOM spec's wording here: it's "fire an event named whatever using ErrorEvent" to fire an instance of ErrorEvent.

What _shoould_ the event fired here be named?  "error" or "processorerror"?

Reply:
Understood. It's `processorerror`.